cpanfile.snapshot

```diff
@@ -3839,6 +3839,19 @@ DISTRIBUTIONS
       IO::WrapTie::Slave 2.110
     requirements:
       ExtUtils::MakeMaker 0
+  IPC-Run-0.94
```

Comment:
Do you know what pulled in IPC::Run?  I don't see it as a dep for Test::RequiresInternet and I don't see you using it anywhere else.